restructuring for shared use PyPy / CPython

wlav opened the pull request:
take out the net dependency
add back plugins/TVirtualStreamerInfo
repackage using loader (from python) and entry-points (for genreflex)
remove a level of indirection
make bdist_wheel universal
rename PyPy-cppyy-backend to cppyy-backend

wlav committed:
take out the net dependency
add back plugins/TVirtualStreamerInfo
repackage using loader (from python) and entry-points (for genreflex)
remove a level of indirection
make bdist_wheel universal

wlav commented:
The checks fail b/c of the renaming PyPy-cppyy-backend -> cppyy-backend. I don't know how to fix those.

antocuni commented:
are you sure to have included all the commits you intended to?
The PR mentions a lot of features but I can see only one commit with modifications to just `_genreflex.py` and `loader.py`

wlav committed:
take out the net dependency
add back plugins/TVirtualStreamerInfo
remove a level of indirection
make bdist_wheel universal
rename PyPy-cppyy-backend to cppyy-backend

wlav commented:
Ouch, yes, only the newly added files were there, not the changes. now fixed.
